account_asset_disposal/models/account_asset_category.py

```diff
@@ -0,0 +1,16 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 Tecnativa - Antonio Espinosa
+# Copyright 2017 Tecnativa - Pedro M. Baeza
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import models, fields
+
+
+class AccountAssetCategory(models.Model):
+    _inherit = "account.asset.category"
+
+    account_loss_id = fields.Many2one(
+        comodel_name="account.account", string="Loss Account",
+        domain=[('type', '=', 'other')],
```

Comment:
internal_type instead of type

Reply:
Ouch, I have forgotten to remove that one. I have removed the other domain in the wizard.

Reply:
Changed